src/python/system/environment.py

```diff
@@ -151,23 +151,13 @@ def get_asan_options(redzone_size, malloc_context_size, quarantine_size_mb,
     remove_key('LSAN_OPTIONS')
     asan_options['detect_leaks'] = 0
 
-  # FIXME: Support container overflow on all platforms.
-  if bot_platform == 'LINUX':
-    asan_options['detect_container_overflow'] = 1
-  else:
-    # Broken on multiple platforms. E.g. Android tracked in b/25228125.
+  # FIXME: Support container overflow on Android.
+  if bot_platform == 'ANDROID':
     asan_options['detect_container_overflow'] = 0
 
-  # Stack use-after-return.
-  # FIXME: Support Stack UAR on all platforms.
-  if (bot_platform in ['LINUX', 'MAC'] and
-      'detect_stack_use_after_return=' not in asan_options):
```

Comment:
detect_stack_use_after_return= does not work, this is now a dict and we later override it 

  # Additional options. These override the defaults.
  additional_tool_options = get_value('ADDITIONAL_%s_OPTIONS' % tool_name)
  if additional_tool_options:
    tool_options.update(_parse_memory_tool_options(additional_tool_options))